tox.ini

```diff
@@ -462,7 +462,7 @@ deps =
   psycopg2 ~= 2.8.4
   aiopg >= 0.13.0, < 1.3.0
   sqlalchemy ~= 1.4
-  redis ~= 3.3.11
+  redis ~= 3.5
```

Comment:
celery tests are failing with redis 3.3 so had to upgrade

Reply:
curious , why did celery tests fail with redis 3.3?

Reply:
We pinned redis but did not pin celery. Latest version of celery/kombu passes extra params to redis which 3.3.x did not accept.